node/libs/roles/src/proto/attester.proto

```diff
@@ -28,10 +28,12 @@ message Signed {
 }
 
 message PublicKey {
+  // The name is wrong, it should be secp256k1.
   optional bytes bn254 = 1; // required
 }
 
 message Signature {
+  // The name is wrong, it should be secp256k1.
```

Comment:
just add a new field and deprecate the old one. Since it is not deployed anywhere yet, we don't need to maintain support for bn254.